tasks/configure-jenkins.yml

```diff
@@ -26,47 +36,8 @@
     group: "{{ jenkins_config_group }}"
   when: jenkins_include_secrets
 
-- name: get token for jenkins api
-  uri:
-    url: '{{ jenkins_url }}:{{ jenkins_port }}/crumbIssuer/api/xml?xpath=concat(//crumbRequestField,":",//crumb)'
-    return_content: yes
-    status_code: 200,404
-  register: jenkins_token
-
-- name: Install plugins if crumb is enabled
-  uri:
-    url: "{{ jenkins_url }}:{{ jenkins_port }}/pluginManager/installNecessaryPlugins"
-    method: POST
-    headers:
-      Content-Type: "text/xml"
-      Jenkins-Crumb: "{{ jenkins_token.content.split(':')[1] | default('noCrumb') }}"
-    body: "<jenkins><install plugin=\"{{ item }}@latest\" /></jenkins>"
-    status_code: 200,302
-  with_items: "{{ jenkins_plugins }}"
-  when: jenkins_token.status == 200
-
-- name: Install plugins if crumb is disabled
-  uri:
-    url: "{{ jenkins_url }}:{{ jenkins_port }}/pluginManager/installNecessaryPlugins"
-    method: POST
-    headers:
-      Content-Type: "text/xml"
-    body: "<jenkins><install plugin=\"{{ item }}@latest\" /></jenkins>"
-    status_code: 200,302
-  with_items: "{{ jenkins_plugins }}"
-  when: jenkins_token.status == 404
-
-
-- name: Custom plugins are installed
-  copy:
-    src: "{{ item }}"
-    dest: "{{ jenkins_home }}/plugins/"
-    owner: "{{ jenkins_config_owner }}"
-    group: "{{ jenkins_config_group }}"
-    mode: 0644
-  with_items: "{{ jenkins_custom_plugins }}"
-  when: jenkins_custom_plugins is defined
-
-- wait_for:
-    path: "{{ jenkins_home }}/plugins/{{ item }}.jpi"
-  with_items: "{{ jenkins_plugins }}"
+- name: set permissions on secrets dir
+  file:
+    path: "{{ jenkins_home }}/secrets"
+    mode: 0700
```

Comment:
I tried doing that, but it set `0700` permissions on all the files *inside* `secrets` and not the directory itself. I'm a bit new to Ansible though, so there's a good chance that I could be missing something there. ;)